frontend/src/App.scss

```diff
@@ -126,27 +126,6 @@
     padding-right: 1.2rem;
   }
 
-  //by default highlight required fields in blue.
```

Comment:
technically this is used by the manage users page, but on that page required field labels are also highlighted in red so I'm fine with this change.

Reply:
I see . That's why I was reluctant to remove global form styling. We need to file a tech-debt ticket to harmonize styling for all forms